apiserver/facades/client/spaces/move_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package spaces_test
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/juju/juju/apiserver/params"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/names.v3"
+
+	"github.com/juju/juju/apiserver/facades/client/spaces"
+	"github.com/juju/juju/apiserver/facades/client/spaces/mocks"
+)
+
+func (s *SpaceTestMockSuite) TestAPIMoveSubnetsSuccess(c *gc.C) {
```

Comment:
I will do a follow up patch the re-arranges there suites. There should be a single base suite and a dedicated suite per model-operation-based method, rather than a giant suite in _spaces_test.go_